_officers/hackutd-17-neha.md

```diff
@@ -1,8 +1,8 @@
 ---
 name: "Neha Rode"
-role: "Director of HackUTD"
-category: board
-filter: "board,hackutd"
+role: "HackUTD Marketing Lead"
```

Comment:
@Neha-Rode you might want to update the officer DB lol, looks like it still says you're the HackUTD Marketing Lead 🙃

Reply:
Updated :))